lib/attributor/types/model.rb

```diff
@@ -141,6 +141,13 @@ def validate(context=Attributor::DEFAULT_ROOT_CONTEXT)
 
         errors.push *sub_attribute.validate(value, sub_context)
       end
+      unless self.class.requirements.empty?
```

Comment:
Can the `self.class.requirements` ever be `nil`, or can we ensure it's always at least an empty array? 

That'd let us cut out this check entirely.